src/it/scala/com.rbmhtechnology.calliope/TransactionalEventProducerSpec.scala

```diff
@@ -404,6 +405,22 @@ class TransactionalEventProducerSpec extends KafkaSpec with MustMatchers with Ty
             .expectNextN(2).map(e => e.value().payload) must contain inOrder(Event("1", "agg1"), Event("2", "agg2"))
         }
       }
+      "stop after being restarted" in {
+        val (topic, consumer) = topicConsumer()
+        val writer = runTransactionalEventProducer(topic, WritableEventStore(), settings)
+
+        whenReady(Future.sequence(Vector(
```

Comment:
if I use `flatMap` the consecutive messages will only be sent after the previous ones were processed, the idea is to send them in short succession to test the `stash()`ing behaviour